python/lsst/daf/butler/transfers/_yaml.py

```diff
@@ -262,6 +272,16 @@ def __init__(self, stream: IO, registry: Registry):
                 element = self.registry.dimensions[data["element"]]
                 RecordClass: Type[DimensionRecord] = element.RecordClass
                 self.dimensions[element].extend(RecordClass(**r) for r in data["records"])
+
+                if data["element"] == "visit" and migrate_visit_system:
+                    # Must create the visit_system_membership records.
+                    element = self.registry.dimensions["visit_system_membership"]
+                    RecordClass = element.RecordClass
+                    self.dimensions[element].extend(
+                        RecordClass(instrument=r["instrument"], visit_system=r["visit_system"], visit=r["id"])
+                        for r in data["records"]
+                    )
```

Comment:
I think we need a migration here for `visit_definition`, too, right, since that had its columns change?

Maybe it's worked because `insertDimensionData` is ignoring columns it doesn't recognize?  (Either that or maybe we don't have test coverage.)

Reply:
It drops unknown columns on the floor.